.golangci.reference.yml

```diff
@@ -2734,13 +2681,22 @@ linters:
     - unparam
     - unused
     - usestdlibvars
-    - varcheck
     - varnamelen
     - wastedassign
     - whitespace
     - wrapcheck
     - wsl
     - zerologlint
+    - deadcode # Deprecated
```

Comment:
what is output will be if I enable deprecated linter?

won't people be confused when they want to enable the deprecated linter?

It looks like it’s now impossible to say for sure whether the linter is deprecated, but it can be enabled or it’s deprecated and removed at all

Reply:
> what is output will be if I enable deprecated linter?

the same as before.

> won't people be confused when they want to enable the deprecated linter?

The deprecation means users should stop using it.

> It looks like it’s now impossible to say for sure whether the linter is deprecated, but it can be enabled or it’s deprecated and removed at all

I can add a message (`reason`) inside the Noop to explain the behavior.